examples/Demo/FluentUI.Demo.Client/Documentation/Components/Tooltip/FluentTooltip.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Tooltip
+route: /Tooltip
+---
+
+# Tooltip
+
+A `FluentTooltip` displays additional information about another component. The information is displayed above and near the target component.
```

Comment:
The information is displayed near the target component.

By using Positioning it can also be placed below etc.